### PR TITLE
updating bistable_perception

### PR DESCRIPTION
**Pull request by PoloOnTheGo**
Please review

### CODE REVIEW

File: study_assets_root/bistablePerception/mainTaskVideoStimuli.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
   <video id="bistableVideo" class="video-js vjs-default-skin" preload="auto" width="640" height="264" autoplay="true"
     loop="true" data-setup='{"example_option":true}'></video>
   <p id="feedbackKeyPress"></p>
-  <p id="quit">Quit Study</p>
+<!--  <p id="quit">Quit Study</p>-->
```
Comment (kristian-lange-tw) [inline study_assets_root/bistablePerception/mainTaskVideoStimuli.html]: Don't leave commented out code, it confuses the users

**Comment by kristian-lange-tw** [comment]
Please don't commit your _.idea_ folder (_study_assets_root/bistablePerception/.idea_). It only has some use for people who use Intellij and most JATOS users do not use it. The HTML file looks fine :) .

File: study_assets_root/bistablePerception/mainTaskVideoStimuli.html (hunk 1 above)
Comment (PoloOnTheGo) [reply study_assets_root/bistablePerception/mainTaskVideoStimuli.html]: sorry, I missed it :(. I will fix the same.

**Comment by PoloOnTheGo** [comment]
I will remove the .idea folder :)

**Comment by kristian-lange** [comment]
Actually I just tried to import the zip file into my JATOS and it's not working. How did you get the zipped study? Although it's possible to zip them yourself you can let JATOS do this: just click on 'Export' in the study page. 

And another point: Although the _bistablePerception_ folder in _study_assets_root_ looks all fine, the zipped folder still has the _.idea_ folder in it and additionally some Mac OS specific files like _.DS_Store_ in it. Is it possible to remove those files before you export the study. Or remove them from the zip file?

**Comment by PoloOnTheGo** [comment]
oh, I zipped the folder that I committed, let me then try the export option, but then I fear, it might take the idea folders :( let me try downloading visual studio otherwise

**Comment by kristian-lange** [comment]
Yes Visual Study Code is a good idea. Or export it, then open the zip and remove the files (this is comfortably possible in Linux - not sure about MacOS though).

**Comment by PoloOnTheGo** [comment]
yeah, in mac, vs is a little pain to work with. Let me try it out and commit tomorrow morning.

**Comment by kristian-lange** [comment]
Btw those Mac files don't hurt the study, it still works - they just might be confusing to users

**Pull request merged**